beginner_source/blitz/neural_networks_tutorial.py

```diff
@@ -44,19 +44,19 @@ class Net(nn.Module):
 
     def __init__(self):
         super(Net, self).__init__()
-        # 1 input image channel, 6 output channels, 3x3 square convolution
-        # kernel
+        # 입력 이미지 채널 1개, 출력 채널 6개, 3x3의 정사각 컨볼루션 행렬
+        # 컨볼루션 커널 정의
         self.conv1 = nn.Conv2d(1, 6, 3)
         self.conv2 = nn.Conv2d(6, 16, 3)
-        # an affine operation: y = Wx + b
-        self.fc1 = nn.Linear(16 * 6 * 6, 120)  # 6*6 from image dimension
+        # 아핀 연산: y = Wx + b
```

Comment:
문서 상에 처음으로 등장하는 용어라, 아래와 같은 식은 어떨까요?
```suggestion
        # 아핀(affine) 연산: y = Wx + b
```

Reply:
좋습니다! 추가하였습니다 👍